README.md

```diff
@@ -6,22 +6,32 @@
 
-Rancher is an open source container management platform built for organizations that deploy containers in production. Rancher makes it easy to run Kubernetes everywhere, meet IT requirements, and empower DevOps teams.
+Rancher is an open source container management platform built for organizations that deploy containers in production.
+Rancher makes it easy to run Kubernetes everywhere, meet IT requirements, and empower DevOps teams.
 
 > Looking for Rancher 1.6.x info? [Click here](https://github.com/rancher/rancher/blob/master/README_1_6.md)
 
 ## Latest Release
+
 * v2.7
-  * Latest - v2.7.5 - `rancher/rancher:v2.7.5` / `rancher/rancher:latest` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.7.5).
-  * Stable - v2.7.5 - `rancher/rancher:v2.7.5` / `rancher/rancher:stable` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.7.5).
+    * Latest - v2.7.5 - `rancher/rancher:v2.7.5` / `rancher/rancher:latest` - Read the full
+      release [notes](https://github.com/rancher/rancher/releases/tag/v2.7.5).
+    * Stable - v2.7.5 - `rancher/rancher:v2.7.5` / `rancher/rancher:stable` - Read the full
+      release [notes](https://github.com/rancher/rancher/releases/tag/v2.7.5).
 * v2.6
-  * Latest - v2.6.13 - `rancher/rancher:v2.6.13` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.6.13).
-  * Stable - v2.6.13 - `rancher/rancher:v2.6.13` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.6.13).
+    * Latest - v2.6.13 - `rancher/rancher:v2.6.13` - Read the full
+      release [notes](https://github.com/rancher/rancher/releases/tag/v2.6.13).
+    * Stable - v2.6.13 - `rancher/rancher:v2.6.13` - Read the full
+      release [notes](https://github.com/rancher/rancher/releases/tag/v2.6.13).
 * v2.5
-  * Latest - v2.5.17 - `rancher/rancher:v2.5.17` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.5.17).
-  * Stable - v2.5.17 - `rancher/rancher:v2.5.17` - Read the full release [notes](https://github.com/rancher/rancher/releases/tag/v2.5.17).
+    * Latest - v2.5.17 - `rancher/rancher:v2.5.17` - Read the full
+      release [notes](https://github.com/rancher/rancher/releases/tag/v2.5.17).
+    * Stable - v2.5.17 - `rancher/rancher:v2.5.17` - Read the full
+      release [notes](https://github.com/rancher/rancher/releases/tag/v2.5.17).
 
-To get automated notifications of our latest release, you can watch the announcements category in our [forums](http://forums.rancher.com/c/announcements), or subscribe to the RSS feed `https://forums.rancher.com/c/announcements.rss`.
+To get automated notifications of our latest release, you can watch the announcements category in
+our [forums](http://forums.rancher.com/c/announcements), or subscribe to the RSS
+feed `https://forums.rancher.com/c/announcements.rss`.
```

Comment:
Why'd all this change, is this unrelated ? Seems like there was a max line setting somewhere. I would prefer to revert all this, as it is less readable IMO.

Reply:
I needed to make a small update to the file. When I saved it, Goland ran the default Markdown formatter, and I chose to keep the changes. Right, the max line is there and line breaks between headers and text, which I think is good, no? It's easier to read raw Markdown, since you don't need to scroll to the right all the time, but it's rendered the same...

Reply:
I suppose since it renders the same it doesn't matter. IMO readability is hit and miss, and better left up to the writer. For example the _"please refer to our Rancher Documentation"_ sentence is now on two lines instead of one. Same for "Please see installation requirements" or "open source libraries and projects". Could you just increase the line length ? If not I would prefer to revert.